### ISSUE TITLE
Extension doesn't start and shows no response

### ISSUE THREAD

**Issue by nforss** [spakin/SimpInkScr#131]
Hi, and thanks for what seems like a very useful extension!

I've installed the extension according to the readme, and it shows up in the Render submenu on my Windows 10 machine. When I select the extension, however, it doesn't open. In fact, I see no kind of reaction from Inkscape at all.

Any idea what could be wrong, or how to get more info about the problem? I can't seem to find nor activate any Inkscape logging or output, so I don't really have any idea where to even start.

The version I'm running is Inkscape 1.3.2 (091e20e, 2023-11-25, custom).

I appreciate any help.

**Comment by spakin** [comment]
Wow, that's a weird problem.  Let's get the simple cases out of the way first:

- Have you installed other extensions, and do they work?
- Have you restarted Inkscape and tried again?

Assuming you installed Simple Inkscape Scripting by downloading the code from GitHub, one thing you can try is to exit Inkscape, remove the Simple Inkscape Scripting directory, go back into Inkscape, and install Simple Inkscape Scripting using Inkscape's Extension Manager (*Extensions* → *Manage Extensions…*).  Maybe that will do something different enough to make the extension work properly for you.

**Comment by nforss** [comment]
I tried the Extensions Manager approach, but didn't get far. Trying to open the Manager itself yields the same result, i.e. absolutely nothing. I then tried some of the other, built in extensions as well. Same thing - no reaction at all.

I think we can conclude that my problem has to do with Inkscape itself and not your extension. I apologise for the bother, but thank you for your response! 

The issue can probably be closed as 'irrelevant' or something.
